env_manager/tests/test_manager.py

```diff
@@ -10,11 +10,26 @@
 from env_manager.manager import Manager
 
 
-# TODO: Add 'venv' and 'micromamba' backends
 BACKENDS = [
-    ("mamba", "python", "mamba", None),
-    ("venv", "pip", "packaging", None),
-    ("conda-like", "python", "packaging", os.environ.get("ENV_BACKEND_EXECUTABLE")),
+    (
+        "venv",
+        "pip",
+        ["packaging"],
+        ["packagingg"],
+        (
+            "Could not find a version that satisfies the requirement packagingg",
+            "WARNING: Skipping packagingg as it is not installed",
+        ),
+        None,
+    ),
+    (
+        "conda-like",
+        "python",
+        ["packaging"],
+        ["packagingg"],
```

Comment:
Quick question because I don't understand it: why does `packagingg` (double g at the end) appears here?

Reply:
That is to test when someone could try to install or uninstall a package that either doesn't exists or is not installed. You can see the messages related with the return of install and uninstall processes for that package bellow. Probably is clearer for the `venv` case since the parameters for the error /warning that pip shows mentions `packagingg`?

Reply:
Oh, I see! I thought it was a spelling mistake or something. In that case I'd suggest using a package name such as `foo` or `foobar` to convey more clearly that this is a made up name.